gr-uhd/lib/rfnoc_rx_radio_impl.cc

```diff
@@ -129,5 +140,74 @@ void rfnoc_rx_radio_impl::set_iq_balance(const std::complex<double>& correction,
     return d_radio_ref->set_rx_iq_balance(correction, chan);
 }
 
+void rfnoc_rx_radio_impl::issue_stream_cmd(const ::uhd::stream_cmd_t& cmd,
+                                           const size_t chan)
+{
+    if (chan == ALL_CHANS) {
+        for (size_t ch = 0; ch < d_radio_ref->get_num_output_ports(); ch++) {
+            d_radio_ref->issue_stream_cmd(cmd, ch);
+        }
+        return;
+    }
+    d_radio_ref->issue_stream_cmd(cmd, chan);
+}
+
+void rfnoc_rx_radio_impl::enable_rx_timestamps(const bool enable, const size_t chan)
+{
+    return d_radio_ref->enable_rx_timestamps(enable, chan);
+}
+
+
+/******************************************************************************
+ * Command handlers
+ *****************************************************************************/
+void rfnoc_rx_radio_impl::_cmd_handler_stream_cmd(const pmt::pmt_t& cmd_p,
+                                                  int chan_i,
+                                                  const pmt::pmt_t& msg)
+{
+    ::uhd::stream_cmd_t cmd(::uhd::stream_cmd_t::STREAM_MODE_START_CONTINUOUS);
+    const std::string stream_mode_str = pmt::write_string(
+        pmt::dict_ref(cmd_p, pmt::mp("stream_mode"), pmt::mp("start_cont")));
+    if (stream_mode_str == "start_cont") {
+        // All set already
+    } else if (stream_mode_str == "stop_cont") {
+        cmd.stream_mode = ::uhd::stream_cmd_t::STREAM_MODE_STOP_CONTINUOUS;
+    } else if (stream_mode_str == "num_done") {
+        cmd.stream_mode = ::uhd::stream_cmd_t::STREAM_MODE_NUM_SAMPS_AND_DONE;
+    } else if (stream_mode_str == "num_more") {
+        cmd.stream_mode = ::uhd::stream_cmd_t::STREAM_MODE_NUM_SAMPS_AND_MORE;
```

Comment:
I like the directness of this, but maybe, since I also like how you moved things into `cmd_keys.h`, do the same with a `std::unordered_map<std::string, uhd::stream_cmd_t>`, assuming this might be something we also want to be able to document explicitly. But this is 100% just me throwing around ideas.